latest-langs

```diff
@@ -36,6 +36,7 @@ constant %paths = (
     'Lua'          => 'en.wikipedia.org/wiki/Lua_(programming_language)',
     'Nim'          => 'en.wikipedia.org/wiki/Nim_(programming_language)',
     'OCaml'        => 'en.wikipedia.org/wiki/OCaml',
+    'Odin'         => 'github.com/odin-lang/Odin/releases/latest',
```

Comment:
I wonder if they'd seen versioning approaches that are more standard, less extraterrestrial.

Reply:
They will switch to semantic versioning when v1.0 is reached: https://www.github.com/odin-lang/Odin/issues/887